portfolio/src/main/webapp/index.html

```diff
@@ -3,17 +3,55 @@
   <head>
     <meta charset="UTF-8">
     <title>My Portfolio</title>
+    <!-- <meta name="viewport" content="width=device-width, initial-scale=1.0"> -->
     <link rel="stylesheet" href="style.css">
+    <!-- <script src="https://kit.fontawesome.com/a076d05399.js" crossorigin="anonymous"></script> -->
     <script src="script.js"></script>
   </head>
   <body>
+    <nav>
+        <input type="checkbox" id="check">
+        <label for="check" class="checkbtn">
+            <i class="fas fa-bars"></i>
+        </label>
+        <label class="logo">Brandocean ⚡️</label>
+        <ul>
+            <li><a class="active" href="#">Home</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Projects</a></li>
+            <li><a href="#">Experience</a></li>
+            <li><a href="#">Contact</a></li>
+            <li><a href="#">Feedback</a></li>
+            <li><a href="Experiments.html">Experiments</a></li>
+        </ul>
+    </nav>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Brandon's Portfolio</h1>
+      <img class="center" src="/images/Brandocean.jpg" width="300" height="300"/>
```

Comment:
[3.1.4](https://google.github.io/styleguide/htmlcssguide.html#Multimedia_Fallback): Provide alternative contents for multimedia.

You should specify the [`alt`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-alt) attribute.